xformers/ops/fmha/dispatch.py

```diff
@@ -31,7 +31,7 @@ def _get_use_fa3() -> bool:
 
 def fa3_available() -> bool:
     has_cuda = torch.version.cuda is not None
-    is_90a = has_cuda and torch.cuda.get_device_capability() >= (9, 0)
+    is_90a = has_cuda and torch.cuda.get_device_capability() == (9, 0)
```

Comment:
actually I forgot to change it - but fa3 now supports ampere. Could you update it here? Thanks!
```suggestion
    is_90a = has_cuda and (8, 0) <= torch.cuda.get_device_capability() <= (9, 0)
```